include/cantera/kinetics/Reaction.h

```diff
@@ -16,6 +16,7 @@
 #include "Falloff.h"
 #include "InterfaceRate.h"
 #include "PlogRate.h"
+#include "TwoTempPlasmaRate.h"
```

Comment:
I believe that at the moment, the headers for individual reaction rate specializations are included via `Reaction.h`. From that perspective, `TwoTempPlasmaRate.h` was an outlier. Other than that, reaction rate headers are only included in `ReactionRateFactory.cpp`.

Reply:
With the exception of the top-level headers (`cantera/thermo.h` etc.), the rest of the Cantera header files are not meant to be shortcuts to including many other headers. The reason so many `ReactionRate` headers were included here previously was only because that was necessary, since the derived `Reaction` types included members of those types.

If the abstraction that we've created in the `ReactionRate` class is robust, it should never be necessary to include the specific implementation headers anywhere except for `ReactionRateFactory.cpp` and classes that derive directly from them. You can see the parallel with the `ThermoPhase` types, which are by and large included only `ThermoFactory.cpp`.

Reply:
This makes sense. In this case, I believe all specialization headers should be removed.